Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -849,6 +849,10 @@ theorem tendsto_norm_div_self (x : E) : Tendsto (fun a => ‖a / x‖) (𝓝 x)
   simpa [dist_eq_norm_div] using
     tendsto_id.dist (tendsto_const_nhds : Tendsto (fun _a => (x : E)) (𝓝 x) _)
 
+@[to_additive]
+theorem tendsto_norm_div_self_nhdsGE (a : E) : Tendsto (‖· / a‖) (𝓝 a) (𝓝[≥] 0) :=
```

Comment:
nit: can you use `x` instead of `a` to match the naming used in the very similar `tendsto_norm_div_self`? Similarly, I think for local consistency it would be better to avoid `·`.